src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,83 @@
-""" This is a dummy example """
+import requests
```

Comment:
This is the file from the first excercise right? AFAIK you already closed a pull request with it, what is it doing here?

Reply:
Hey Guille! I am using the same branch for all PR, I think that's why there are previous files. Should I created a new branch for every homework?

Reply:
Yes, the idea is that we have separate branches for each exercise. In fact, in more general terms, the normal flow of working with GitHub is to open a branch to work on a certain feature (in this case, a specific exercise). Once the feature is implemented, we open a PR, and once the PR is approved, we merge the whole branch. This means that it's rare to see a branch with multiple PRs.

Reply:
perfect, I will take that into consideration while doing the next PR. I will upload it today. Thank you!